### PR TITLE
Notification revamp

### PR DESCRIPTION
**Pull request by BreadJS**
## Abstract

As planned, the request to revamp the notifications to give it a better look and better interface for users to interact with.

### CODE REVIEW

File: tests/unit/alert.spec.js [hunk 1]
```diff
@@ -75,7 +75,13 @@ describe('createAlert function', () => {
         const level = 'info';
         createAlert(level, message);
 
-        expect(createAlertSpy).toHaveBeenCalledWith(level, message, undefined);
+        expect(createAlertSpy).toHaveBeenCalledWith(
```
Comment (panleone) [inline tests/unit/alert.spec.js:78]: Improve unit test coverage by testing the two new fields
```
actionName,
actionFunc,
```

**Review by panleone — CHANGES_REQUESTED** [review]
code looks good and works as expected. My only comment is that unit tests should be updated

**Review by panleone — DISMISSED** [review]
tACK 6aea10e46b81014f00a1bd8f49a7260378a293fe

File: scripts/alerts/Alert.vue [hunk 2]
```diff
@@ -25,14 +27,41 @@ const icon = computed(() => {
...
         :style="{ opacity: 1 }"
         data-testid="alert"
     >
-        <div class="notifyIcon" :class="{ ['notify-' + level]: true }">
-            <i class="fas fa-xl" :class="{ [icon]: true }"> </i>
+        <div
+            class="notifyBadgeCount"
+            v-if="notificationCount > 1"
+            v-html="notificationCount"
+        ></div>
```
Comment (Duddino) [inline scripts/alerts/Alert.vue:38]:
```suggestion
        >{{notificationCount}}</div>
```

**Review by Duddino — APPROVED** [review]
tACK, with a very minor nit

**Pull request merged**